bugsnag/__init__.py

```diff
@@ -2,6 +2,7 @@
 from bugsnag.notification import Notification
 from bugsnag.event import Event
 from bugsnag.client import Client
+from bugsnag.breadcrumbs import BreadcrumbType, Breadcrumb
```

Comment:
I'm not sure whether we should be exposing these classes, but I think this is useful for type annotations